lib/elixometer.ex

```diff
@@ -240,10 +240,15 @@ defmodule Elixometer do
   Updates a timer metric. If the metric doesn't exist, it will be created and
   subscribed to the default reporter.
 
-  The time units default to *microseconds*, but you can pass in a unit of
-  `:millis` and the value will be converted.
+  The time units default to *microseconds*, but you can also pass in any of
+  the units in [`System.time_unit.t`](https://hexdocs.pm/elixir/System.html#t:time_unit/0),
```

Comment:
I had to fiddle with the syntax a little, but I was able to get `t:System.time_unit/0` to link correctly.